### ISSUE TITLE
Lack of reply button on a TextNote with quoted TextNote 

### ISSUE THREAD

**Issue by Giszmo** [monlovesmango/astral#25]
TextNote

```
{
  "id": "fba0722266ff7c252c72d5ace7d516611bb62c2dc74e4fdacafdfb6a36b975e4",
  "pubkey": "32e1827635450ebb3c5a7d12c1f8e7b2b514439ac10a67eef3d9fd9c5c68e245",
  "created_at": 1655070996,
  "kind": 1,
  "tags": [
    [
      "e",
      "ec25b9c7ff8fa8ccdc7d2e3bfa06df82448a88c40212c6d19bce4a6f747b736b"
    ],
    [
      "e",
      "528458f58b76fcf91351e5f3794778016f95b762047b3b9fb1a8bb6444e80971"
    ],
    [
      "p",
      "3efdaebb1d8923ebd99c9e7ace3b4194ab45512e2be79c1b7d68d9243e0d2681"
    ],
    [
      "p",
      "8355095016fddbe31fcf1453b26f613553e9758cf2263e190eac8fd96a3d3de9",
      "wss://nostr-relay.wlvs.space"
    ],
    [
      "p",
      "46fcbe3065eaf1ae7811465924e48923363ff3f526bd6f73d7c184b16bd8ce4d"
    ],
    [
      "p",
      "09e935f7c01fda340051a4700cfb9dde533202bdf56808f68cafef6bae07a5bd"
    ],
    [
      "e",
      "4c712ebfc251f02f9af0ab39e4c4b4fa9427a76e5826e67138dc048a41cba22c"
    ]
  ],
  "content": "#[6]",
  "sig": "2f3886e583dd44bcb52a8f4555cbe938eb56f23378341bb436f4ac78b18f3fe16ddf7d8c7bd813f964e05811c424ae094dbe413aca96967d436978d9e399345a"
}
```

quotes Note

```
{
  "id": "4c712ebfc251f02f9af0ab39e4c4b4fa9427a76e5826e67138dc048a41cba22c",
  "pubkey": "32e1827635450ebb3c5a7d12c1f8e7b2b514439ac10a67eef3d9fd9c5c68e245",
  "created_at": 1654962253,
  "kind": 1,
  "tags": [],
  "content": "Haha sorry everyone accidentally clicking the boost button, I will have a confirmation button in the next testflight release",
  "sig": "8277df96de140c6bc0b831019b2d84155608827f89b584e773ad95c19208356ddd89d18aa9d2002eca7febcb04c8fc3c469f756924cbee6818ecc483ae0d7ca4"
}
```

I tried to reply to the former but there was no reply button. But there was a reply button for the latter.

![Screenshot from 2022-06-12 21-25-24](https://user-images.githubusercontent.com/96378/173264522-1b3a0b94-2f4b-46d5-b7a9-33657e3a43c7.png)

I would expect it to be reverse: only a reply button for the new Note. No reply button for the quoted Note.

But to make matters worse, it's now no reply at all according to Astral! Just a top-level Note quoting two Notes.

**Comment by monlovesmango** [comment]
so I just tested this and seem to be able to reply to a quoted note (like you were doing in the screenshot). will keep this open for a little bit in case I can replicate the error.

as far as the reply button on the repost, I removed it intentionally. I want to encourage people to reply to original post, rather than having a separate side conversation on a repost. especially since a repost itself doesn't provide any original content and doesn't add to the discourse (whereas with a quote post you at least are adding some original thoughts so I left the ability to reply). I'm open to changing my mind if others feel strongly and have good rational.
